.github/minimal-rmf-jazzy/Dockerfile

```diff
@@ -0,0 +1,40 @@
+ARG ROS_DISTRO=jazzy
+ARG BASE_IMAGE=docker.io/ros:$ROS_DISTRO-ros-core
+FROM $BASE_IMAGE
+
+# These branches are only required for Jazzy, as some required messages are not
+# backported to be released.
+ARG INTERNAL_MSGS_BRANCH=dd2951debfe53d0391896ecb9ddf07f027af05e9
+ARG BUILDING_MAP_MSGS_BRANCH=e26cf73ec7b1f61bbd450a4f85450e0db20d6c72
```

Comment:
I'm not sure if I understand this, `minimal-rmf/Dockerfile` is changed to install rmf from binary, it should work for jazzy as well. The comments suggest that we are using messages incompatible with jazzy, even if there is no practical differences, it still invalidates this as a jazzy build.

Reply:
as mentioned in the conversation below